src/JsonSchema/Uri/UriRetriever.php

```diff
@@ -40,19 +40,19 @@ class UriRetriever implements BaseUriRetrieverInterface
      *
      * @param UriRetrieverInterface $uriRetriever
      * @param string $uri
-     * @return bool|void
+     * @return bool
      */
     public function confirmMediaType($uriRetriever, $uri)
     {
         $contentType = $uriRetriever->getContentType();
 
         if (is_null($contentType)) {
             // Well, we didn't get an invalid one
-            return;
+            return true;
```

Comment:
@FlorianSW thank you so much for your feedback. This is exactly my concern.

As you pointed out, `null` is falsy. Perhaps returning `false` here is the solution as it would be compliant with the `bool` return type and also be compatible with previous control-flow usage, although that would not be true for `=== null` or `is_null` comparisons.